Sources/Shared/VimeoClient.swift

```diff
@@ -253,7 +289,10 @@ extension VimeoClient {
                 }
             }
         )
-        task?.resume()
+        
+        if startImmediatelly {
```

Comment:
So if false is passed in, `task` would need to be triggered from the `RequestToken`, yeah?

Reply:
That's right. And that's why `RequestToken` has been modified to also allow for `resume()`. I think this gives better control of when to run the request on the calling site while maintaining backwards compatibility with the current behavior (previously we didn't have an option because of the AFNetworking methods we were using).